theme.css

```
@@ -312,6 +312,39 @@
   font-weight: 500;
 }
 
+/* 按钮 loading 图标：TDesign 默认 LoadingIcon 是不完整圆弧，按钮里观感像异形图标；统一改为标准圆形旋转器。 */
+#app .t-button .t-icon-loading {
+  width: 1em;
+  height: 1em;
+  box-sizing: border-box;
+  border: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)